lib/error.js

```diff
@@ -4,14 +4,12 @@ var util = require('util'),
     Controller = require('hmpo-form-controller');
 
 function getArgs(type, args) {
-    if (type === 'maxlength') {
-        return { maxlength: args[0] };
-    } else if (type === 'minlength') {
-        return { minlength: args[0] };
-    } else if (type === 'exactlength') {
-        return { exactlength: args[0] };
-    } else if (type === 'past') {
+    if (type === 'past') {
         return { age: args.join(' ') };
+    } else if (args !== undefined) {
```

Comment:
Looks like these arguments ultimately come from the getMessage() method on FormError. Which is exposed to the outside world. So I suppose we should have some validation going on either in here or the getMessage() method.